### DISCUSSION TITLE
Help to make the teleportation pay

### DISCUSSION THREAD

**Discussion by Letraqueur** [Zrips/CMI#2428] [Q&A]
Hello,

I am looking for a way to make the teleportation (/ tp and / tpa minimum) pay with the money of the economy part of the plug-in.

Sur d'autres forums, on m'a dit que c'était possible avec ce super plug-in en modifiant le fichier config. Unfortunately, I was not told how to do it, and I'm pretty bad at it ^^

Can you help me or advise me?

PS: Sorry for my English, I'm french and speak very bad english, so I use a translation site.

Thank you in advance,
Letraqueur

**Comment by Letraqueur** [comment]
I tested this on a friend's advice, but it does not work. is there an error in the construction?

> CustomAlias:
>   tppos:
>     Cmds:
>     - CostsMoney:15#! tppos $1-
>   tpa:
>     Cmds:
>     - moneycost:15#! tpa $1
>   tpo:
>     Cmds:
>     - moneycost:15#! tpo $1
>   tp:
>     Cmds:
>     - moneycost:15#! tp $1-

**Comment by DrakePork** [comment]
Try moneycost:15! without the #, as the # is for cancelling all commands in a customAlias, not just the one the moneycost is on.

**Comment by Letraqueur** [comment]
Good evening,
I just tried, without  success. If you have another idea of ​​the problem, I'm listening :)

**Comment by Letraqueur** [comment]
Good evening,
I managed to work around the problem by making payments to an Admin account.
Only, I have this message that appears when I want to make the payments ...

"Thie user is not accepting payments from others players"

Does anyone have an idea of ​​why?
I specify that I have 2 accounts admin and that I have the concern only on only one ...
That the problem remains even if I remove the operator power.
Even if I remove it from permission groups ...
That it worked shortly before.
The only change is a nickname change for the account in: "Bank_Server" on the mojang quote.

An idea ?

**Comment by Letraqueur** [comment]
ho the idiot ^^
thank you very much

**Comment by Letraqueur** [comment]
Hello,

I take advantage of your lights for one last question ^^
I have this message in my console at server launch : 

> [14:21:06] [Server thread/INFO]:   Failed to load Locale_EN.yml! Backup have been saved into plugins/CMI/FileBackups/2019-10-11 14-21-06 Locale_EN.yml

The beginning of my config.yml file is fine though:

> # Language file you want to use
> Language: EN

An idea of ​​the reason for this bug? I did not move the file, I just translated a line ...

ligne 706 :   noMoney: '&cVous n'avez pas assez d'argent!'

[Photo](https://image.noelshack.com/fichiers/2019/41/5/1570797575-bug.png)

**Comment by DrakePork** [comment]
You probably need to add two ' in n'avez and d'argent

```
ligne 706 : noMoney: '&cVous n''avez pas assez d''argent!'
```

**Comment by Letraqueur** [comment]
Thank you very much, the worst is that I knew it but forgot to do it.
Sorry for the inconvenience and thank you again :)

**Comment by CloudeLecaw** [comment]
would be nice if you close the ticket ;)
